### DISCUSSION TITLE
Error on tests

### DISCUSSION THREAD

**Discussion by MarceloHoffmeister** [spatie/laravel-activitylog#854] [Q&A] [answered]
I'm trying to implement one feature for this package and having one error on tests.
The error says `Error: Call to undefined method <Test Class>::getAnnotations()`. Is that normal?
I saw in the badges tests failing so this generate my doubt.

**Comment by Gummibeer** [comment]
Thanks for reporting - I'm on it.
We've used an PHPunit internal method which was dropped in v9.5 or so.
If you install phpunit:9.3 you should be able to run the tests properly.
You can achieve this by calling `composer install --prefer-lowest`.
Or you wait till I've fixed the test-suite.

https://github.com/spatie/laravel-activitylog/tree/fix-test-suite

**Comment by Gummibeer** [comment, marked as answer]
https://github.com/spatie/laravel-activitylog/releases/tag/3.17.0

**Reply by MarceloHoffmeister** [reply]
Great! I'm found some solutions for that on web but preferred to wait hear from collaborators. I'm going to try your solution for now. Thank you.
